### DISCUSSION TITLE
hashes/signature and website link for unpacked-linux.tar.gz

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#5878] [Ideas]
It's nice that you are also offering an unpacked version.

Also publishing the hashes of the files and signing it cryptographically would be nice. And also add a link on the website...

**Comment by charlag** [comment]
Unpacked version is primarily for flathub build.

Other builds are already signed in multiple ways, please see https://github.com/tutao/tutanota/blob/master/buildSrc/installerSigner.js

**Reply by ghost** [reply]
the appimage in linux is repackaged as an arch package or as a deb.  (some of us, prefer that the package manager keeps track of what's installed)

If the unpacked version was properly signed and hashed, we could be using that instead.

Regardless of what end users do with it, it should still be properly hashed and signed like the others.
